<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongWidget</class>
 <widget class="QWidget" name="SongWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>56</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>931</width>
    <height>56</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>01</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>img</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_collection">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>collection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Like">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>12</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../matter.qrc">
     <normaloff>:/icon/img/Like_1.png</normaloff>:/icon/img/Like_1.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>music_name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_composer">
      <property name="text">
       <string>music_composer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../matter.qrc"/>
 </resources>
 <connections/>
</ui>
